src/testdrive/action/mod.rs

```diff
@@ -21,13 +21,16 @@ use regex::{Captures, Regex};
 use repr::strconv;
 
 use crate::error::{Error, InputError, ResultExt};
-use crate::parser::{Command, PosCommand};
+use crate::parser::{Command, PosCommand, SqlExpectedResult};
 
 mod avro_ocf;
 mod file;
 mod kafka;
 mod sql;
 
+/// 12.7 seconds (12700 milliseconds)
+const DEFAULT_TIMEOUT: u64 = 12700;
```

Comment:
If we use a `Duration` here, you won't need the comment, because `Duration::from_millis(12_700)` is nice and self documenting.

Reply:
Also, as below, perhaps `DEFAULT_SQL_TIMEOUT`?